README.md

```diff
@@ -38,8 +36,23 @@ nextflow run UPHL-BioNGS/Grandeur -profile docker --fastas <path to fastas>
 
 WARNING : "Grandeur" will automatically grab any fastq files in `workflow.launchDir + '/reads'`, fasta files in `workflow.launchDir + '/fastas'`, and gff files in `workflow.launchDir + '/gff'`. This is a **feature** of the workflow.
 
-## Grandeur wouldn't be possible without:
+## Wiki sections
+
+The README got too long, so it's been moved to a [wiki](https://github.com/UPHL-BioNGS/Grandeur/wiki). There are several covered topics including:
+- [Installation](https://github.com/UPHL-BioNGS/Grandeur/wiki/Installation)
+- [Usage](https://github.com/UPHL-BioNGS/Grandeur/wiki/USAGE)
+  - [Stadalone](https://github.com/UPHL-BioNGS/Grandeur/wiki/USAGE#standalone)
```

Comment:
Change Stadalone to Standalone.

Reply:
How embarrassing! Thank you for catching that!